src/main/kotlin/dev/stalla/parser/DateParser.kt

```diff
@@ -17,8 +17,10 @@ import java.util.Locale
  * format to produce the intended date object.
  */
 @InternalApi
+
 internal object DateParser {
 
+    @Suppress("MagicNumber")
     private val dayOfWeek = mapOf(
         1L to "Mon",
         2L to "Tue",
```

Comment:
Felt stupid to introduce constants here.

Reply:
Absolutely :)